File: flask_app/models/review.py
class Review:
    def __init__(self, id, book_id, user_id, comment, status, star, created_at, updated_at, deleted_at):
        self.id = id
        self.book_id = book_id
        self.user_id = user_id
        self.comment = comment
        self.status = status
        self.star = star
        self.created_at = created_at
        self.updated_at = updated_at
        self.deleted_at = deleted_at

File: react_app/models/review.js
class Review {
    constructor(
      id,
      bookId,
      userId,
      comment,
      status,
      star,
      createdAt,
      updatedAt,
      deletedAt
    ) {
      this.id = id;
      this.bookId = bookId;
      this.userId = userId;
      this.comment = comment;
      this.status = status;
      this.star = star;
      this.createdAt = createdAt;
      this.updatedAt = updatedAt;
      this.deletedAt = deletedAt;
    }
  }
  
  export default Review;
  